File: app/src/main/java/github/fushaolei/entity/BaseResponse.java
package github.fushaolei.entity;

import okhttp3.MediaType;
import okhttp3.ResponseBody;
import okio.BufferedSource;

/**
 * @Auther: fushaolei
 * @datetime: 2021/2/11
 * @desc:
 */
public class BaseResponse {
    private int code;
    private String msg;

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    @Override
    public String toString() {
        return "BaseResponse{" +
                "code=" + code +
                ", msg='" + msg + '\'' +
                '}';
    }
}

File: app/src/main/java/github/fushaolei/entity/BaseResponse.kt
package github.fushaolei.entity

/**
 * @Auther: fushaolei
 * @datetime: 2021/2/11
 * @desc:
 */
class BaseResponse {
    var code = 0
    var msg: String? = null
    override fun toString(): String {
        return "BaseResponse{" +
                "code=" + code +
                ", msg='" + msg + '\'' +
                '}'
    }
}